Containers/recording/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM python:3.11.3-alpine3.18
+
+COPY --chmod=775 start.sh /usr/bin/start.sh
+COPY --chmod=664 recording.conf /etc/recording.conf
+
+RUN set -ex; \
+    apk add --no-cache \
+        ca-certificates \
+        tzdata \
+        bash \
+        ffmpeg \
+        libpulse \
+        bind-tools \
+        netcat-openbsd \
+        git \
+        wget \
+        shadow; \
+    # xvfb firefox chromium chromium-chromedriver?
+    # apk add --no-cache geckodriver --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing; \
+    useradd --system recordning; \
+    git clone --recursive https://github.com/nextcloud/spreed --branch v16.0.3 /src; \
+    mv -v /src/recording/pyproject.toml /src/recording/src/pyproject.toml; \
+    python3 -m pip install /src/recording/src; \
+    rm -rf /src; \
```

Comment:
```suggestion
    mv -v /src/recording/pyproject.toml /src/recording/src/pyproject.toml; \
    python3 -m pip install /src/recording/src; \
    rm -rf /src/recording; \
```

Reply:
why? git clones it to /src

Reply:
I see, my mistake. Lets leave it like this then